tests/e2e/vmgroups_test.go

```diff
@@ -66,7 +66,7 @@ func (vg *VmGroupTest) SetUpSuite(c *C) {
 		log.Printf(out)
 	}
 	// Create the test VM group1
-	cmd = adminconst.CreateVMgroup + vgTestVMgroup1 + " --default-datastore " + vg.config.Datastores[0]
+	cmd = adminconst.CreateVMgroup + vgTestVMgroup1 + " --default-datastore '" + vg.config.Datastores[0] + "'"
```

Comment:
This works and I'm assuming that any command line (need to check) done this way for the datastore name _should_ work.

_docker volume inspect snapvol2@'sharedVmfs-0'_
[
    {
        "Driver": "vsphere",
        "Labels": null,
        "Mountpoint": "/mnt/vmdk/snapvol2@sharedVmfs-0",
        "Name": "snapvol2@sharedVmfs-0",
        "Options": {},
        "Scope": "global",
        "Status": {
            "access": "read-write",
            "attach-as": "independent_persistent",
            "capacity": {
                "allocated": "15MB",
                "size": "100MB"
            },
            "clone-from": "None",
            "created": "Sun Aug  6 13:01:34 2017",
            "created by VM": "worker1-VM1.0",
            "datastore": "sharedVmfs-0",
            "diskformat": "thin",
            "fstype": "ext4",
            "status": "detached"
        }
    }
]